.github/workflows/kinesis-test.yml

```diff
@@ -0,0 +1,20 @@
+name: Integration tests - Kinesis
+
+on: push
+
+jobs:
+  kinesis_no_data_lost:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: coursier/cache-action@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Initialize the resources and the collector and send data to the collector
+        run: '"$PWD"/integration/init.sh kinesis "$PWD"/integration/docker-compose.yml integration.hocon "$PWD"/.github/workflows/integration_tests/no-data-loss/data.txt'
```

Comment:
Here you can see how the initialisation script works. This allows us to locally spin up the required resources so the tests can be ran locally.